templates/telecommunications/index.html

```diff
@@ -0,0 +1,207 @@
+      <ul class="p-list">
+        <li><a href="/cloud/juju">Find out more about Juju&nbsp;&rsaquo;</a></li>
+        <li><a href="/server/maas">Find out more about MAAS&nbsp;&rsaquo;</a></li>
+      </ul>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip--light is-bordered">
+  <div class="row">
+    <div class="col-8">
+      <h2>NFVI cloud build</h2>
+      <p>Canonical&rsquo;s Foundation Cloud Build for Telcos is a pre-tested, pre-integrated OpenStack cloud built using a proven reference architecture and tailored to the needs of CSPs. Think of it as a fixed price, NFV-ready cloud, deployed by Canonical on your premises in just a few weeks.</p>
+      <p><a href="/cloud/foundation-cloud">Find out more<span class="u-off-screen"> about NFVI cloud build</span>&nbsp;&rsaquo;</a></p>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip--light is-bordered">
+  <div class="row">
+    <div class="col-8">
+      <h2>A telco-grade managed cloud</h2>
+      <p>Canonical&rsquo;s Foundation Cloud Build for Telcos is a pre-tested, pre-integrated OpenStack cloud built using a proven reference architecture and tailored to the needs of CSPs. Think of it as a fixed price, NFV-ready cloud, deployed by Canonical on your premises in just a few weeks.</p>
+      <p><a href="/cloud/foundation-cloud">Find out more<span class="u-off-screen"> about Canonical&rsquo;s Foundation Cloud Build</span>&nbsp;&rsaquo;</a></p>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip--light is-bordered">
+  <div class="row">
+    <div class="col-8">
+      <h2>Smart edge: Internet of Things, switches and <abbr title="Customer-Premises Equipment">CPE</abbr>s</h2>
+      <p>Ubuntu is the Operating System of choice for the smart edge. From software-defined top of the rack switch to enterprise CPE, from home IoT gateways to digital signage, Canonical offers secure OS and appstore solutions to maximise post sales revenue from all your smart devices.</p>
+      <p><a href="/internet-of-things">Find out more<span class="u-off-screen"> about smart edge</span>&nbsp;&rsaquo;</a></p>
+    </div>
+  </div>
+</section>
+
+<div class="p-strip--accent is-deep">
+  <div class="row">
+    <div class="col-12">
+      <blockquote class="p-pull-quote--accent">
+        <p>We&rsquo;re reinventing how we scale by becoming simpler and modular, similar to how applications have evolved in cloud data centers. Open source and OpenStack innovations represent a unique opportunity to meet these requirements and Canonical&rsquo;s cloud and open source expertise make them a good choice for AT&T.</p>
+        <cite class="p-pull-quote__citation">Toby Ford, Assistant Vice President of Cloud Technology, Strategy and Planning at AT&T</cite>
+      </blockquote>
+    </div>
+  </div>
+</div>
+
+<section class="p-strip is-bordered">
+  <div class="row">
+    <div class="col-12">
```

Comment:
Maybe make the logo more narrow so there is not one on its own.

Reply:
Done